docs/source/models/intervention_models/neonatal/cpap_intervention.rst

```diff
@@ -112,6 +112,24 @@ where :math:`p(\text{RDS})` is the probability of dying from Preterm with RDS in
 
 When we fill in the location-specific values for delivery facility rates, CPAP coverage, relative risk of mortality with CPAP access, and mortality probability (which is also age-specific), this becomes a system of two linear equations have two unknown (p_RDS_w and p_RDS_wo), which we can solve analytically.
 
+We can derive the PAF and RR of lack of access to CPAP from these quantities as follows::
+
+  # TODO: replace these stand-in values
+  # with appropriate artifact draws
+  p_RDS = .1
+  p_home = .5
+  p_BEmONC = .1
+  p_CEmONC = .4
+  p_CPAP_BEmONC = 0.075
+  p_CPAP_CEmONC = 0.393
+  RR_CPAP = 0.53
+
+  p_RDS_w = ... # solve system of equations from previous section
```

Comment:
This is overwritten immediately on line 130?

Reply:
I was doing more of a constraint programming than procedural programming thing here...  line 130 is really the definition of PAF_no_CPAP, but I thought it would be clearer to write out the way I did, instead of solving to put PAF_no_CPAP on the left hand side.

Reply:
This feels discordant with your response to the other comment -- once you comment out 130 this can be seen as procedural again, unless I am missing something

Reply:
I'm going to merge the change commenting out line 130, and you can ignore my thinking out loud about constraint programing if it is not helpful.